mail_delivery_robot/config/map.csv

```diff
@@ -0,0 +1,35 @@
+﻿,Loeb,Southam,TunnelJunction,Library,Azrieli/Tory,Steacie,Richcraft,UC,Canal,Nicol,Mackenzie/Minto,Frontenac,ResComms,CTTC,Robertson,,,
+Loeb1,DOCK,U-TURN,U-TURN,U-TURN,U-TURN,LEFT,LEFT,U-TURN,U-TURN,U-TURN,U-TURN,U-TURN,U-TURN,U-TURN,U-TURN,,,Make sure to go TOWARDS THE BEACON
+Loeb2,DOCK,RIGHT,RIGHT,RIGHT,RIGHT,U-TURN,U-TURN,RIGHT,RIGHT,RIGHT,RIGHT,RIGHT,RIGHT,RIGHT,RIGHT,,,
+Southam1,FORWARD,DOCK,U-TURN,U-TURN,U-TURN,FORWARD,FORWARD,U-TURN,U-TURN,U-TURN,U-TURN,U-TURN,U-TURN,U-TURN,U-TURN,,,
+Southam3,U-TURN,DOCK,FORWARD,FORWARD,FORWARD,U-TURN,U-TURN,FORWARD,FORWARD,FORWARD,FORWARD,FORWARD,FORWARD,FORWARD,FORWARD,,,

```

Comment:
You should remove the 3 empty commas the end of each row here.

Reply:
Done! Thanks for catching that